Source/WebKit/WebProcess/WebCoreSupport/WebPermissionController.h

```diff
@@ -61,6 +59,7 @@ class WebPermissionController final : public CanMakeWeakPtr<WebPermissionControl
     struct PermissionRequest {
         WebCore::ClientOrigin origin;
         WebCore::PermissionDescriptor descriptor;
+        WebCore::PageIdentifier identifier;
```

Comment:
Why a PageIdentifier when we need a WebPageProxyIdentifier?

Reply:
We've gone back to using WebPageProxyIdentifier here--fixed!